proto/cosmos/orm/v1alpha1/query.proto

```diff
@@ -0,0 +1,131 @@
+syntax = "proto3";
+
+package cosmos.orm.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/any.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+// Query is a generic gRPC service for querying ORM data.
+service Query {
+
+  // Get queries an ORM table against an unique index.
+  rpc Get(GetRequest) returns (GetResponse);
+
+  // List queries an ORM table against an index.
```

Comment:
is there a way to do a form of reflection for this? in the same way grpc reflection works?

Reply:
What kind of reflection do you mean?

Reply:
https://github.com/grpc/grpc/blob/master/doc/server-reflection.md

Reply:
The protobuf file descriptors themselves contain all the ORM definitions and the grpc server reflection you linked above allows the client to retrieve these already.

In addition, the app module would eventually allow for querying an app's list of loaded modules (separately from the file descriptor set): https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/app/v1alpha1/query.proto

That probably is enough for clients to do all the reflection they need, but we could add other convenience services later related to the ORM if that makes things simpler